### ISSUE TITLE
After last upgrade error in rule deprecate/import

### ISSUE THREAD

**Issue by crystalfp** [AlexMost/eslint-plugin-deprecate#28]
After plugin upgrade to `"eslint-plugin-deprecate": "^0.8.0"` the following error appears:
```
Oops! Something went wrong! :(

ESLint: 8.44.0

TypeError: Cannot read properties of undefined (reading 'name')
Occurred while linting D:\Projects\VUEIdeaIgniterEnvironment\client\src\services\AddFullDocumentView.ts:24
Rule: "deprecate/import"
    at MemberExpression (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint-plugin-deprecate\dist\rules\import.js:45:41)
    at ruleErrorHandler (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\linter.js:1050:28)
    at D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\node-event-generator.js:297:26)
    at NodeEventGenerator.applySelectors (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\node-event-generator.js:326:22)
    at NodeEventGenerator.enterNode (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\node-event-generator.js:340:14)
    at CodePathAnalyzer.enterNode (D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\code-path-analysis\code-path-analyzer.js:795:23)
    at D:\Projects\VUEIdeaIgniterEnvironment\client\node_modules\eslint\lib\linter\linter.js:1085:32
```
Putting off the `deprecated/import` rule solved the problem. BTW, this problem was absent in the previous version of the plugin.

OS: Windows 11
node: v19.8.1
npm: 9.7.2
eslint: v8.44.0

Thanks for looking!
mario

**Comment by ejfrancis** [comment]
I've just opened a PR to fix this https://github.com/AlexMost/eslint-plugin-deprecate/pull/29

**Comment by AlexMost** [comment]
fix is available here - eslint-plugin-deprecate@0.8.2. Thanks for the contribution!
